workspaces/lightspeed/plugins/lightspeed-backend/.prettierrc.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+/** @type {import("@ianvs/prettier-plugin-sort-imports").PrettierConfig} */
+module.exports = {
+  ...require('@spotify/prettier-config'),
+  plugins: ['@ianvs/prettier-plugin-sort-imports'],
+  importOrder: [
+    '^react(.*)$',
+    '',
+    '^@backstage/(.*)$',
+    '',
+    '<THIRD_PARTY_MODULES>',
+    '',
+    '^@janus-idp/(.*)$',
```

Comment:
I didn't migrated the prettier config also if I'm a big fan of sorted imports. :)

If you want keep it, maybe update this line to `@backstage-community/`?

Reply:
Other plugins use `@red-hat-developer-hub`

Reply:
In this repo, of course ;) :+1: